src/state/apis/snapshot/snapshot.ts

```diff
@@ -0,0 +1,95 @@
+import { createApi } from '@reduxjs/toolkit/dist/query/react'
+import { type AccountId, fromAccountId } from '@shapeshiftoss/caip'
+import { isEvmChainId } from '@shapeshiftoss/chain-adapters'
+import snapshot from '@snapshot-labs/snapshot.js'
+import axios from 'axios'
+import { bn, bnOrZero } from 'lib/bignumber/bignumber'
+import { findClosestFoxDiscountDelayBlockNumber } from 'lib/fees/utils'
+
+import { BASE_RTK_CREATE_API_CONFIG } from '../const'
+import type { Strategy } from './validators'
+import { SnapshotSchema, VotingPowerSchema } from './validators'
+
+type SnapshotVotingPowerArgs = AccountId[]
+type FoxVotingPowerCryptoBalance = string
+
+const SNAPSHOT_SPACE = 'shapeshiftdao.eth'
+
+export const snapshotApi = createApi({
+  ...BASE_RTK_CREATE_API_CONFIG,
+  reducerPath: 'snapshotApi',
+  endpoints: build => ({
+    getStrategies: build.query<Strategy[], void>({
+      keepUnusedDataFor: Number.MAX_SAFE_INTEGER, // never refetch these
+      queryFn: async () => {
+        const query = `
+          query {
+            space(id: "shapeshiftdao.eth") {
+              strategies {
+                name
+                network
+                params
+              }
+            }
+          }
+        `
+        // https://hub.snapshot.org/graphql?query=query%20%7B%0A%20%20space(id%3A%20%22shapeshiftdao.eth%22)%20%7B%0A%20%20%20%20strategies%20%7B%0A%20%20%20%20%20%20name%0A%20%20%20%20%20%20network%0A%20%20%20%20%20%20params%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D
+        const { data: resData } = await axios.post(
+          'https://hub.snapshot.org/graphql',
+          { query },
+          { headers: { Accept: 'application/json' } },
+        )
+        try {
+          const { strategies } = SnapshotSchema.parse(resData).data.space
+          return { data: strategies }
+        } catch (e) {
+          console.error('### snapshotApi getStrategies', e)
```

Comment:
prefer no prefix on the logging since (now that logger is gone) we can see it originated here in the stack trace